src/Cardano/Address.hs

```diff
@@ -1,17 +1,24 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE QuasiQuotes #-}
 
+{-# OPTIONS_HADDOCK prune #-}
+
 module Cardano.Address
     ( -- * Address
-      Address (..)
+      Address
     , PaymentAddress (..)
+    , unsafeMkAddress
```

Comment:
wonder if we want to expose it when we have from/to text

Reply:
ah, ok. it is used in style extensively

Reply:
Yes, and it is exposed internally, but not visible from Haddock :)